docs/_includes/ww-admin-tasks-content.md

```diff
@@ -0,0 +1,240 @@
+
+<img alt="Checking property collection details" src="/assets/img/ww-user-guide/WasteWorks-select-property-dropdown-menu.png" />
+
+</div>
+
+<div class="admin-task" markdown="1" id="creating-reports-subscriptions-on-behalf-of-user">
+
+### Creating requests/reports/subscriptions on behalf of a resident
+
+<span class="admin-task__permissions">Permissions required: User must be marked
+as staff; optionally, one or more of ‘Create reports/updates on a user's behalf’, ‘Create reports/updates as anonymous user’ and ‘Create reports/updates as the council’ can be ticked.</span>
+
+If a resident makes a report or request by phone or in person, staff members with the appropriate permissions can add it to WasteWorks on their behalf. 
+
+Staff should make a new report/request or subscription just as a member of the public would (see the resident’s experience).
+
+**The process for setting up, renewing or cancelling subscriptions for users will differ depending on the payment provider you are using.** 
+
+</div>
+
+<div class="admin-task" markdown="1" id="cancelling-subscriptions">
+
+### Cancelling subscriptions on behalf of a resident
+
+<span class="admin-task__permissions">Permissions required: User must be marked
+as staff; optionally, one or more of ‘Create reports/updates on a user's behalf’, ‘Create reports/updates as anonymous user’ and ‘Create reports/updates as the council’ can be ticked.</span>
+
+Staff users are able to cancel subscriptions on behalf of residents in the same way the resident would do this - see The Resident’s experience. 
+
+**The process for setting up, renewing or cancelling subscriptions for users will differ depending on the payment provider you are using.** 
+
+</div>
+
+<div class="admin-task" markdown="1" id="creating-response-templates">
+
+### Creating and editing response templates
+
+<span class="admin-task__permissions">Permissions required: User must be marked as staff; “Add/edit response templates” must be ticked..</span>
+
+Staff users with the appropriate permissions may create and edit pre-written responses, so that they can be applied to reports quickly and easily.
+
+<img alt="Checking property collection details" src="/assets/img/ww-user-guide/WasteWorks-response-templates.png" />
+
+#### Creating a template
+
+Click on ‘Templates’ in the admin menu. You will see a table of any existing templates. Click on the button below, marked ‘New template’.
+
+Common template responses deal with statuses such as: Report received; Collection scheduled; Container ordered; Report under investigation.
+
+Templates can also be created to stop residents from reporting, for example, a missed collection during certain time periods. These templates allow you to display up-to-date information on the 'Your bin days' page to explain why a collection may have been delayed (eg changes to servives over a Bank Holiday).
```

Comment:
Sorry if wasn't clear - these aren't templates to stop residents from reporting, they're to use instead of the backend's default messaging for a particular collection's resolution. e.g. Bromley's backend has a task resolution of "Con - Nappies", (Con means Contaminated) and the template for that resolution code is then "Your waste could not be collected because it contained the wrong materials. [...]".